projects/NAB/results/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!final_results.json
```

Comment:
I think it did? I'm intentionally including `final_results.json` and excluding everything else.